src/client/Microsoft.Identity.Client/Platforms/iOS/EmbeddedWebview/MsalAuthenticationAgentUIViewController.cs

```diff
@@ -53,12 +53,12 @@ public override void ViewDidLoad()
         protected WKWebView PrepareWKWebView()
         {
             WKWebViewConfiguration wkconfg = new WKWebViewConfiguration() { };
-#if !IS_APPCENTER_BUILD
```

Comment:
I think this flag also appears in the some of the YAML build defintions..

Reply:
yes, it is still needed for the broker test app that we create for the broker team to test for us. However, these are no longer needed in any of the shipped product code.